trackers-whitelist.txt

```diff
@@ -129,6 +129,8 @@
 ||gallery.mailchimp.com^$image
 ! unbreak videos on foxnews
 ||acdn.adnxs.com/prebid/*/prebid.js$domain=foxnews.com|foxbusiness.com
+! unblock site search toolbars powered by Elastic's Swiftype
+||*.api.swiftype.com
```

Comment:
Our APIs use both `*.api.swiftype.com` and `api.swiftype.com`, so need to whitelist wider or have two rules: App Search API (`*.api.swiftype.com`) and Site Search API (`api.swiftype.com`)

Reply:
Good catch, updated. :+1: